### ISSUE TITLE
Design day temperature and other schedules do not interpolate to timestep

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5778]
## Issue overview

_From user:_
In the attached IDF file, I am using Schedule:Day:Interval objects to define summer designday drybulb, wetbulb, beam and diffuse solar radiation profiles. 

In the Schedule:Day:Interval objects, I have set the "Interpolate to Timestep" to "Yes". However, from the simulation result, I can see that the output values were not interpolated to each simulation time step (10 min), rather they are hourly step functions, see the attached plots.

![diffusesolar](https://cloud.githubusercontent.com/assets/4991075/16892089/32bde232-4acd-11e6-8461-799a5ce0bfa7.PNG)

![directsolar](https://cloud.githubusercontent.com/assets/4991075/16892090/38091d92-4acd-11e6-8318-cade192acf5a.PNG)

![tdb](https://cloud.githubusercontent.com/assets/4991075/16892092/3b617412-4acd-11e6-9701-cb453225a87c.PNG)

![twb](https://cloud.githubusercontent.com/assets/4991075/16892094/40cb22fe-4acd-11e6-8fa8-fddbd0ff9ba2.PNG)

### Workaround
Per comments in #6330, this is technically working as designed, but it will be changed to work as expected (after v8.8).  Until then, a workaround is to use minute-by-minute schedules using Schedule:Day:Interval, Schedule:Day:List, or Schedule:Compact.

### Details

Some additional details for this issue (if relevant):
- Platform (Win64)
- Version of EnergyPlus (v8.5.0)
- helpdesk ticket number 11368
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [ ] Defect file added (EnergyPlusDevSupport\DefectFiles)
- [ ] Ticket added to Pivotal for defect (development team task)
- [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
Comments from a developer emeritus:
I think there are two sort of problems here. (not sure I have the time to track them down). The schedule:interval is not designed to do what he wants. I tried putting in the Untils on the 1/2 hour but that didn't really help. I used schedule:hourly for some and that wasn't a solution either -- so basically, either the fill of the schedule values is not set up to "interpolate" over the timesteps like weather data is or the design day creation doesn't do that.
